### DISCUSSION TITLE
Can there be an option for exporting it as Fountain markup too. It would be truly helpful. Fountain export and preview

### DISCUSSION THREAD

**Discussion by rokfrog** [vkbo/novelWriter#792] [Ideas]
I see this Novel writing software is pretty good. 
Since there are many softwares for screenwriting, but this Novelwriter seems very good contentor for Screenwriting if it worked for screenwriting too.

Something like, say Character, "Hello How are you", he smirks. 
if say we write like above than the editor automatically puts the Character centered and given a dialogue beneath with He smirks inside parenthetical. There are 4 key elements to screenwriting and how it aligns within 1 min= 1 page concept. Fountain markup exporting would be great inclusion

**Comment by vkbo** [comment]
Hi, and thanks for your suggestion. You're not the first to ask for Fountain support, so I've looked into it in the past. The syntax used by novelWriter is not compatible with Fountain, so unfortunately it will not work without major modifications.

I saw a talk at a Qt Dev conference last week about an application for Screenwriting called [Scrite](https://github.com/teriflix/scrite). Maybe that is a better fit?
